src/endpoint_resolver.rs

```diff
@@ -116,6 +116,7 @@ impl MomentoEndpointsResolver {
 }
 
 #[cfg(test)]
+#[allow(clippy::unwrap_used)]
```

Comment:
I think this can be removed now.

Reply:
Thanks, missed this one! Addressed in https://github.com/momentohq/client-sdk-rust/pull/76/commits/5a8b158f5f3cc26c7a04aedb4f5c9cb6d51e387e